UnityMcpBridge/UnityMcpServer~/src/tools/manage_asset.py

```
@@ -86,15 +91,15 @@ def _coerce_int(value, default=None):
             "pageSize": page_size,
             "pageNumber": page_number
         }
-        
+
         # Remove None values to avoid sending unnecessary nulls
         params_dict = {k: v for k, v in params_dict.items() if v is not None}
 
         # Get the current asyncio event loop
         loop = asyncio.get_running_loop()
         # Get the Unity connection instance
         connection = get_unity_connection()
```

Ruff (0.13.1)

Local variable `connection` is assigned to but never used

Remove assignment to unused variable `connection`

(F841)